pkg/ddl/executor.go

```diff
@@ -109,7 +111,7 @@ type Executor interface {
 	CreateSchema(ctx sessionctx.Context, stmt *ast.CreateDatabaseStmt) error
 	AlterSchema(sctx sessionctx.Context, stmt *ast.AlterDatabaseStmt) error
 	DropSchema(ctx sessionctx.Context, stmt *ast.DropDatabaseStmt) error
-	CreateTable(ctx sessionctx.Context, stmt *ast.CreateTableStmt) error
+	CreateTable(ctx context.Context, sctx sessionctx.Context, stmt *ast.CreateTableStmt) error
```

Comment:
can we pass `context.Background()` like others, so no need to pass this separate `ctx